internal/generate/packagemanifest/packagemanifest.go

```diff
@@ -17,37 +17,63 @@ package packagemanifest
 import (
 	"errors"
 	"fmt"
-	"io"
+	"io/ioutil"
 	"path/filepath"
 	"sort"
 
+	"sigs.k8s.io/yaml"
+
 	apimanifests "github.com/operator-framework/api/pkg/manifests"
 	"github.com/operator-framework/api/pkg/validation"
 	log "github.com/sirupsen/logrus"
 
 	genutil "github.com/operator-framework/operator-sdk/internal/generate/internal"
-	"github.com/operator-framework/operator-sdk/internal/generate/packagemanifest/bases"
 )
 
 const (
 	// File extension for all PackageManifests written by Generator.
 	packageManifestFileExt = ".package.yaml"
 )
 
+// GPkgManenerator is an implementation of the Generator interface
+type generator struct{}
+
+// NewGenerator returns a new generator object
+func NewGenerator() Generator {
+	return generator{}
+}
+
+// Generator is an interface that specifies the Generate methods
+// to generate and write various package manifests
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . Generator
```

Comment:
Can you add `go generate ./...` to the `generate` make target? That way all counterfeits will always be regenerated.